Dockerfile

```diff
@@ -27,12 +30,23 @@ RUN curl -sL https://deb.nodesource.com/setup_13.x | bash - \
    && apt-get clean
 
 RUN yarn install
+
 # Have to set SECRET_KEY_BASE here to arbitrary string, otherwise task doesn't run
-RUN SECRET_KEY_BASE=valuenotactuallyused rails assets:precompile
+RUN SECRET_KEY_BASE=valuenotactuallyused bundle exec rails assets:precompile
+
+# Run the application as user `moj` (created in the base image)
```

Comment:
is this comment still relevant given that the base image has changed from `ministryofjustice/ruby:2.6.2-webapp-onbuild` to `ruby:2.6.2-stretch`
...and on a related note, what is the reason for changing the base image?

Reply:
Yes the comment is wrong. The base image didn't work, and turned out to be deprecated and undocumented